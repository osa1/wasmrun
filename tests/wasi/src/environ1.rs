fn main() {
    unsafe {
        main_();
    }
}

unsafe fn main_() {
    let (n_vars, buf_size) = wasi::environ_sizes_get().unwrap();
    println!("n_vars={}, buf_size={}", n_vars, buf_size);
}
