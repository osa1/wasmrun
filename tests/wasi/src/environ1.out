n_vars=0, buf_size=0
